build.sbt

```diff
@@ -21,11 +19,8 @@ libraryDependencies ++= Seq(
   "com.typesafe" % "config" % "1.3.0",
   "org.typelevel" %% "cats" % "0.7.2",
   "com.typesafe.akka" %% "akka-actor" % akkaV,
-  "io.spray" %% "spray-can" % sprayV,
-  "io.spray" %% "spray-routing" % sprayV,
-  "io.spray" %% "spray-client" % sprayV,
-  "io.spray" %% "spray-http" % sprayV,
-  "io.spray" %% "spray-json" % downgradedSprayV,
+  "com.typesafe.akka" %% "akka-http-experimental" % akkaV,
+  "com.typesafe.akka" %% "akka-http-spray-json-experimental" % akkaV,
```

Comment:
But yet they're not considering the akka folks have been telling people to jump in with both feet for several months now and it is the officially supported